deployment/deployment.go

```diff
@@ -87,15 +88,19 @@ func List(ws string, all bool) error {
 
 	deployments = r.Data.GetDeployments
 
+	rows := []printutil.TempRow{}
+
 	// Build rows
 	for _, d := range deployments {
 		if all {
 			ws = d.Workspace.Uuid
 		}
-
-		tab.AddRow([]string{d.Label, d.ReleaseName, d.Version, d.Id}, false)
+		row := printutil.TempRow{[]string{d.Label, d.ReleaseName, d.Version, d.Id}, false}
```

Comment:
Let's keep this part simple and limit it to a single call for adding a row to the table vs two calls for adding a row and a third call to add all rows.